src/main/java/com/google/sps/JobModel.java

```diff
@@ -34,29 +34,37 @@
 // Class used to represent a Job in memory, extracting just the 
 // information required by the analysis
 public abstract class JobModel {
-    public static final int SSD = 1;
-    public static final int HDD = 2;
+    public static final Integer SSD = 1;
+    public static final Integer HDD = 2;
     String projectId;
     String name;
     String id;
+
+    // Type can have following values:
+    // - JOB_TYPE_STREAMING
+    // - JOB_TYPE_UNKNOWN
+    // - JOB_TYPE_BATCH
     String type;
     String sdk = null;
+    // sdkSupportStatus ca be found at 
```

Comment:
```suggestion
    // sdkSupportStatus can be found at 
```
Just a tiny spelling mistake.

Reply:
I'll correct this. Thanks!

Reply:
resolved this in https://github.com/googleinterns/step229-2020/pull/15/commits/d93c9abd9592079a818b7ad7396acdd078cc1aab